src/styles/navbar.scss

```diff
@@ -32,6 +32,25 @@
 .show-nav-search{
   visibility: visible;
 }
+.logout {
+  text-align: center;
+	padding: 8px 16px 8px 15px;
+	display: inline-block;
+	color: $white;
+  font-size: 12px;
+  font-weight: bold;
+	background-color: $colorpurple;
+	border-radius: 5px;
+	text-decoration: none;
+	transition: background-color 300ms;
+
+	box-shadow: 0 5px 4px $black25;
+	&:hover {
+		color: $white;
+		background-color: $colorclairvoyant;
+		text-decoration: none;
+	}
+}
```

Comment:
There seem to be indentation issues in this style declaration block. Please resolve.

Reply:
Have you been able to address this?

Reply:
Hello @andela-moe, there are no indentation issue in the style declaration block.